src/meta/src/hummock/compaction/picker/mod.rs

```diff
@@ -32,11 +32,72 @@ use crate::hummock::level_handler::LevelHandler;
 
 #[derive(Default)]
 pub struct LocalPickerStatistic {
-    pub skip_by_write_amp_limit: u64,
-    pub skip_by_count_limit: u64,
-    pub skip_by_pending_files: u64,
-    pub skip_by_overlapping: u64,
+    skip_by_write_amp_limit: bool,
+    skip_by_count_limit: bool,
+    skip_by_pending_files: bool,
+    skip_by_overlapping: bool,
+    skip_by_trivial_move: bool,
```

Comment:
nits: see my comment in src/meta/src/hummock/compaction/mod.rs. We can avoid this boolean and all the related methods if we store a Vec<SkipType> in LocalPickerStatistic.